test-tools/wamr-ide/VSCode-Extension/src/utilities/lldbUtilities.ts

```diff
@@ -0,0 +1,92 @@
+/*
+ * Copyright (C) 2022 Amazon.com Inc. or its affiliates. All rights reserved.
+ * SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+ */
+
+import * as vscode from 'vscode';
+import * as os from 'os';
+import * as path from 'path';
+import * as fs from 'fs';
+import { CheckIfDirectoryExist, DownloadFile, UnzipFile } from './directoryUtilities';
+
+const LLDB_RESOURCE_DIR = "resource/debug";
+const LLDB_OS_DIR_MAP: Partial<Record<NodeJS.Platform, string>> = {
+    "linux": "linux",
+    "darwin": "osx"
+};
+const LLDB_OS_DOWNLOAD_URL_SUFFIX_MAP: Partial<Record<NodeJS.Platform, string>> = {
+    "linux": "x86_64-ubuntu-22.04",
+    "darwin": "universal-macos-latest"
+};
+
+const WamrLLDBNotSupportedError = new Error("WAMR LLDB is not supported on this platform");
```

Comment:
[style] please either use UPPERCASE_SNAKE_CASE (`LLDB_OS_DOWNLOAD_URL_SUFFIX_MAP`) or PascalCase (`WamrLLDBNotSupportedError`)

Reply:
I've opened an issue to setup a linter https://github.com/bytecodealliance/wasm-micro-runtime/issues/1688 so we can avoid this kind of distractions.